src/app/api/user.ts

```diff
@@ -0,0 +1,14 @@
+import { AxiosResponse } from 'axios';
+import instance from './axiosInstance';
+
+const USER = {
+  path: `/users`,
+
+  /** 유저 정보 불러오기 API */
+  async getUserInfo(userId: number): Promise<any> {
```

Comment:
```suggestion
  async getUserInfo(userId: number): Promise<유저정보에 대한 반환 타입> {
```
처음에 구성할 때는 `<any>`로 간단하게 테스트 하고 넘어가도 괜찮지만, 만약 백엔드의 명확한 타입이 명시가 되어있다면, 해당 타입에 맞게 `interface`나 `type`을 구성해서 넣어주면 좋을 것 같습니다.

진행하실 떄, 따로 `types폴더`를 만들어서 진행하면 관리하기 편할겁니다.

Reply:
타입폴더 만들려고 생각중입니다.!